+ ./venv/bin/python3 test_myflask.py
psycopg2.OperationalError: connection to server at "flask-app-postgres-db.cn4wyakgw2cz.us-east-1.rds.amazonaws.com" (172.31.80.219), port 5432 failed: FATAL:  database "flaskapp" does not exist